homeassistant/components/tplink/__init__.py

```diff
@@ -126,3 +186,29 @@ def legacy_device_id(device: SmartDevice) -> str:
     if "_" not in device_id:
         return device_id
     return device_id.split("_")[1]
+
+
+def _get_store(hass: HomeAssistant) -> Store[dict[str, dict[str, str]]]:
+    return Store[dict[str, dict[str, str]]](hass, STORAGE_VERSION, STORAGE_KEY)
```

Comment:
It would slightly cheaper to subclass the store so the type doesn't have to be constructed every time

Reply:
Now the store is persisted by `async_setup` once it is called I don't think this is needed.  Let me know.

Reply:
Store is now dropped